### ISSUE TITLE
[enhancement]: jsonConfig Chips - add hint to use <CR>

### ISSUE THREAD

**Issue by mcm1957** [ioBroker/ioBroker.admin#2870]
### No existing issues.

- [X] There is no existing issue for my request.

### Related problems

none

### Description

When encountering a jsonConfig field of type "CHIPS" I', always wondering why the config cannot be saved after typing in some chars. Its completly OK that a CR is needed to terminate the CHIP Element entry. But this is not obvious.

So I suggest to eveluate whether it is possible (and desired) to show some hint near the CHIPS component link "Press CR to enter" at least if no help is provided by config. I've seen several "error reports" at forum too where user tell that the entered config vanishes after closing (which is OK as they did not press <cr>)

So to summarize:
The functionality is completly OK and nee not and should not changed in any way.
I only want to suggest to add same indocato that a CR is need to continue.

**Comment by Feuer-sturm** [comment]
@mcm1957 can you please check with current beta version if this issue is fixed?

**Comment by mcm1957** [comment]
Hint below line to enter words is shown
